package-lock.json

```diff
@@ -45142,6 +45142,21 @@
         "shallowequal": "^1.1.0"
       }
     },
+    "react-infinite-scroll-component": {
```

Comment:
why is there any diff to package-lock.json?

Reply:
Fixing this now

Reply:
doesn't seem fully fixed